app/views/admin/editions/show/_main.html.erb

```diff
@@ -99,17 +99,23 @@
     <% end %>
   </section>
 
-  <% if @edition.change_note_required? && @edition.change_note %>
+  <% if @edition.change_note_required? %>
     <section class="app-view-edition-summary__section">
       <%= render "govuk_publishing_components/components/heading", {
         text: "Public change note",
         heading_level: 2,
         font_size: "l",
         margin_bottom: 3,
       } %>
-      <p class="govuk-body">
-        <%= @edition.change_note %>
-      </p>
+      <% if @edition.change_note %>
```

Comment:
You could wrap the `if` statement with the `p` tag to avoid repeating it

Reply:
thanks will do